modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PageMemoryStorageIoModule.java

```diff
@@ -38,8 +37,7 @@ public Collection<IoVersions<?>> ioVersions() {
                 TableMetaIo.VERSIONS,
                 TableInnerIo.VERSIONS,
                 TableLeafIo.VERSIONS,
-                TableDataIo.VERSIONS,
-                PartitionMetaIo.VERSIONS
+                TableDataIo.VERSIONS
```

Comment:
So, partition is now outside of table module, but it's aware about "rootId", weird

Reply:
Agree, do I need to do it in this PR?

Reply:
Please no, it's huge already